Dockerfile

```diff
@@ -30,6 +30,7 @@ RUN apt-get update && \
   xvfb && \
   apt-get clean && \
   rm -rf /var/lib/apt/lists/*  && \
+  echo "options(repos = c(CRAN = 'https://cran.rstudio.com/'), download.file.method = 'libcurl')" >> /usr/local/lib/R/etc/Rprofile.site \
```

Comment:
yup, this looks good.  (though we do it as well, dictating `download.file.method` is arguably a bit aggressive, though libcurl should be available on these images and I believe will enable parallel downloads.

Reply:
Thanks for the feedback @cboettig!